Training/strings.cpp

```diff
@@ -17,8 +20,8 @@ int hackerrank::strings()
 
     // swap the first characters
     char t = a[0];
-    a[0] = b[0];
-    b[0] = t;
+    a[0]   = b[0];
```

Comment:
Uhh... I think I was just lining up the `=` signs. In very big code-bases, a fair amount of pruning goes into the visual layout of code. It can help for maintainability and is generally an appreciated practice. Typically a studio will have conventions on when you should align operators and when you should not. In this case I did it for the talking point ;).

If you look at old C _Quake_ code, they may as well have written it in excel- literally tables of code.

Reply:
Wow, that's super strange for me. Coming from a JavaScript world with multiple levels of transpilation and code bases that contain 3 or 4 flavors of language, this level of code prettiness seems unmaintainable.